server/master/src/main/java/org/apache/accumulo/master/MasterTime.java

```diff
@@ -104,7 +105,7 @@ public void run() {
       case UNLOAD_METADATA_TABLETS:
       case UNLOAD_ROOT_TABLET:
         try {
-          zk.putPersistentData(zPath, Long.toString(System.nanoTime() + skewAmount).getBytes(UTF_8),
+          zk.putPersistentData(zPath, Long.toString(System.nanoTime() + skewAmount.get()).getBytes(UTF_8),
```

Comment:
This wasn't previously synchronized. Is there any risk that this will cause additional lock contention with the Master calling `getTime()` on this object? Or does this perhaps a fix a correctness bug? Or neither?

Reply:
It does not seem like this change introduces any lock contention, its now reading from a volatile which seems like an improvement.

Reply:
Okay. I couldn't remember if it grabbed a lock on read, and wasn't in a place I could look at the code. Thanks for checking.